addons/fuse-proxy/Makefile

```
@@ -0,0 +1,22 @@
+.PHONY: vet fmt tidy reviewable
+
+# Default go command
+GO ?= go
+
+# Clean build artifacts
+clean:
+	rm -rf $(BUILD_DIR)
+
+reviewable: fmt vet tidy
+	git diff --exit-code || (echo "Changes detected after static checks. Run 'cd addons/fuse-proxy && make reviewable' and commit the changes." && exit 1)
+
+fmt:
+	$(GO) fmt ./...
+
+vet:
+	$(GO) vet ./...
+
+# Tidy and verify Go dependencies
+tidy:
+	$(GO) mod tidy
+	$(GO) mod verify
```

```suggestion
	$(GO) mod verify

gofumpt:
	gofumpt -l -w .

build-deps:
	$(GO) install mvdan.cc/gofumpt@latest

```
I like gofumpt, we should add it (this does mean a dev would need to run the following command to include `GOBIN` to their `PATH`, like `export PATH=$PATH:$(go env GOPATH)/bin`

---

running `make gofumpt` will produce some diffs I believe, we should commit those too

---

Thanks for adding this!